Formula/qwerty-fr.rb

```diff
@@ -0,0 +1,40 @@
+class QwertyFr < Formula
+  desc "Qwerty keyboard layout with French accents"
+  homepage "https://qwerty-fr.org"
+  url "https://github.com/qwerty-fr/qwerty-fr/releases/download/v0.7/qwerty-fr_0.7_mac.zip"
+  sha256 "34bc8c8f764798ed7e48a3f2404319e09a6d8bfcbef350b25df32387a3199bfa"
+  license "MIT"
+
```

Comment:
It seems this needs to declare a macOS dependency.

Reply:
How can I declare that this formula is macOS-specific?

Reply:
`depends_on :macos`